Refactoring/1033 refactor prediction task.java

chaitraliagashe opened the pull request:
**What's in the PR**
* Refactoring task - PredictionTask.java

**How to test manually**
* Start inception
* Login
* Import Project
* Go to annotation screen
* screen should load the annotations and active learning tab should work

ukp-svc-jenkins commented:
Can one of the admins verify this patch?

reckart commented:
Jenkins, can you test this please?

reckart commented:
@chaitraliagashe the new file is lacking a license header - this causes the build to fail:

```
[WARNING] Files with unapproved licenses:
  /home/svc_jenkins/workspace/INCEpTION (GitHub) (PR builder)/git/inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/util/PredictionUtil.java
```

chaitraliagashe commented:
Hi Richard,
I have reverted the changes related to formatting and added the changes related to refactoring. Formatting changes had affected some other files (pom.xml and few htmls). I have undone them as well. 
Please let me know if this is too confusing.

Regards,
Chaitrali

reckart commented on inception-recommendation/pom.xml:
```diff
@@ -1,253 +1,244 @@
-  Copyright 2017
-  Ubiquitous Knowledge Processing (UKP) Lab
-  Technische Universität Darmstadt
+<!-- Copyright 2017 Ubiquitous Knowledge Processing (UKP) Lab Technische 
```
This file contains lots of whitespace changes that need to be reverted. Please note that XML files are indented using spaces only and using 2 spaces for an indentation.

reckart commented on inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/project/ProjectRecommendersPanel.html:
```diff
@@ -18,11 +18,13 @@
-      <div class="flex-content flex-v-container flex-only-internal-gutter" wicket:id="recommenderEditor"></div>
-    </div>
-  </div>
+	<div class="flex-content flex-v-container flex-gutter">
```
This file contains lots of whitespace changes that need to be reverted. Please note that HTML files are indented using spaces only and using 2 spaces for an indentation.

reckart commented on inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/project/RecommenderEditorPanel.html:
```diff
@@ -18,111 +18,122 @@
-      </div>            
-    </form>
-  </wicket:panel>
+	<wicket:panel>
```
This file contains lots of whitespace changes that need to be reverted. Please note that HTML files are indented using spaces only and using 2 spaces for an indentation.

reckart commented on inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/project/RecommenderListPanel.html:
```diff
@@ -18,18 +18,21 @@
-      </div>            
-    </div>
-  </wicket:panel>
+	<wicket:panel>
```
This file contains lots of whitespace changes that need to be reverted. Please note that HTML files are indented using spaces only and using 2 spaces for an indentation.

reckart commented on inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/sidebar/LearningCurveChartPanel.html:
```diff
@@ -18,15 +18,15 @@
-      <div wicket:id="chart-container"></div>
-    </div>
-  </div>
+	<div class="panel panel-default panel-small">
```
This file contains lots of whitespace changes that need to be reverted. Please note that HTML files are indented using spaces only and using 2 spaces for an indentation.

reckart commented on inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/sidebar/RecommendationSidebar.html:
```diff
@@ -18,45 +18,54 @@
-      </div>
-    </div>
-  </div>
+	<div
```
This file contains lots of whitespace changes that need to be reverted. Please note that HTML files are indented using spaces only and using 2 spaces for an indentation.

reckart commented:
Jenkins, can you test this please?

ukp-svc-jenkins commented:
[![48% (-0.07%) vs master 48%](https://zoidberg.ukp.informatik.tu-darmstadt.de/jenkins//coverage-status-icon/?coverage=0.484&masterCoverage=0.4847)](https://zoidberg.ukp.informatik.tu-darmstadt.de:443/jenkins/job/INCEpTION%20(GitHub)%20(PR%20builder)/2782/)

chaitraliagashe commented:
Hi Richard,
I see you have merged my branch with master and pushed the changes. Is
there anything I need to do now?

If not I will get started on #1069 -
https://github.com/inception-project/inception/issues/1069.
Please change its label/ category to Refactoring as I can not do it and
assign it to me.

Thank you,
Chaitrali

On Thu, Apr 11, 2019 at 11:55 PM UKP OSS Jenkins <notifications@github.com>
wrote:

> [image: 48% (-0.07%) vs master 48%]
> <https://zoidberg.ukp.informatik.tu-darmstadt.de:443/jenkins/job/INCEpTION%20(GitHub)%20(PR%20builder)/2782/>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/inception-project/inception/pull/1060#issuecomment-482242041>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/Ast6yA63mEXlJthgmbCURndf859inqtGks5vf34LgaJpZM4b_BiP>
> .
>


-- 
~Chaitrali

reckart commented:
@chaitraliagashe I have updated the PR, but I didn't manually test yet. Once I have done that, I'll merge.
